news/_posts/core-weekly/2019-05-14-coreweekly-week-19-2019.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 19 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-05-14 16:30:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 6th to Sunday 12th of May 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+In case you missed it, [1.7.6 beta has been released yesterday](http://build.prestashop.com/news/prestashop-1-7-6-0-beta-release/). It's time to test it, to test your modules and themes with it, and [open a new issue](https://github.com/PrestaShop/PrestaShop/issues/new?template=bug_report.md) if needed. Also, of course, your help is welcome to fix issues.
+
+Please add the date to your agenda: [PSD Paris 2019](https://www.prestashop.com/fr/evenements/prestashop-day-paris) will be in a couple of weeks, the 4th of June 2019, at Palais Brongniart, Paris, France.
+
```

Comment:
Maybe explain a bit what it is, why it's interesting to come ?

Reply:
is it better now ?

Reply:
Yes 😲 and that's big news 😄